index.html

```diff
@@ -122,6 +122,541 @@ <h4>LET'S CONNECT</h4>
+                       
+                        </a>
+                   
+                     </li>
+
+                     <li>
+                        
+                        <a href="https://www.angellist.com/">
+                           
+                            <img src="media/ange.png" alt="angellist" class="icono">
+                           
+                        </a>
+                       
+                     </li>
+
+                     <li>
+                         
+                         <a href="https://twitter.com/">
+                            
+                             <img src="media/twit.png" alt="twitter" class="icono" >
+                        
+                         </a>
+
+                     </li>
+
+                    
+
+                     <li>
+                        
+                         <a href="https://medium.com/">
+                            
+                             <img src="media/med.png" alt="medium" class="icono" >
+                        
+                         </a>
+
+                     </li>
+
+                </ul>
+
+             </div>
+
+             <div class="margen grupo-boton ">
+                
+                 <button class="flex-row">Get my resume</button>
+
+             </div>
+
+         </div>
+
+         <div class="acerca-conocimiento">
+
+             <ul class="margen">
+
+                 <li class="conocimiento flex-row primero">
+
+                     <p id="leng">Languages</p>
+                     
+                      <img src="media/abajo.png" alt="flecha" id="sigue2">
+
+                 </li>
+
+                 <ul class="idioma-grupo">
+                    
+                     <li class="flex-row idioma">
+                        
+                         <img src="media/script.png" alt="javascriot">
+                        
+                          <p>JavaScript</p>
+
+                     </li>
+
+                     <li class="flex-row idioma ">
+                        
+                         <img src="media/index.png" alt="html">
+                        
+                         <p>HTML</p>
+
+                     </li>
+
+                     <li class="flex-row idioma">
+                        
+                         <img src="media/style.png" alt="CSS icon">
+                        
+                         <p>CSS</p>
+
+                     </li>
+
+                 </ul>
+
+                 <li class="flex-row conocimiento ">
+
+                     <p id="frameworks">Frameworks</p>
+                    
+                     <img src="media/sigue.png" alt="flecha" id="sigue3">
+
+                 </li>
+
+                 <li class="flex-row conocimiento ">
+
+                     <p>Skills</p>
+                    
+                     <img src="media/sigue.png" alt="flecha" id= "sigue">
+
+                 </li>
+
+             </ul>
+
+         </div>
+
+     </div>
```

Comment:
Kindly ensure you implement these changes :

- Please ensure the container of this section is a `section` tag for a good HTML semantics 😄

Reply:
Done👍